client/src/components/ProductDetails.jsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import RelatedItemsAndComparison from './RelatedItemsAndComparison';
+import QnAWidget from './QnAWidget'
 import ProductOverview from './ProductOverview'
 
 const ProductDetails = function ProductDetails() {
   return (
     <div>
       <ProductOverview />
       <RelatedItemsAndComparison />
+      <QnAWidget id={65631}/>
```

Comment:
instead of hardcoded id, try to use useParams in react-router to get the query parameter of id.